zkvm/docs/zkvm-spec.md

```diff
@@ -866,7 +883,7 @@ Then, the VM executes the current program till completion:
 
 If the execution finishes successfully, VM performs the finishing tasks:
 1. Checks if the stack is empty; fails otherwise.
-2. Checks if the uniqueness flag is set to `true`; fails otherwise.
+2. Checks if the last anchor is present; fails otherwise.
```

Comment:
```suggestion
2. Checks if the last anchor is set to other than ∅; fails otherwise.
```

Reply:
this reads strangely to me. Maybe simply  "checks if the last anchor is set"?

Reply:
```
2. Checks if the last anchor is set; fails otherwise.
```